shared/simplestreams/sort.go

```diff
@@ -19,53 +18,43 @@ func (a sortedImages) Swap(i, j int) {
 }
 
 func (a sortedImages) Less(i, j int) bool {
-	if a[i].Properties["type"] != a[j].Properties["type"] {
-		if a[i].Properties["type"] == "squashfs" {
-			return true
+	for _, prop := range []string{"os", "release", "variant", "serial", "architecture"} {
```

Comment:
@stgraber you mentioned earlier that order was important when comparing these fields, if this is correct, could you add a comment to that effect here please.